pkg/providers/cloudstack/testdata/expected_results_main_md.yaml

```diff
@@ -65,9 +65,11 @@ metadata:
 spec:
   template:
     spec:
-      offering: m4-large
-      template: centos7-k8s-118
-      details:
-        foo: bar
+      offering:
+        id: 
```

Comment:
Lets remove the id so that its not confusing.

Reply:
yes, we have leave it in in our current template-md implementation. To remove it, we need to put conditional statement in template-cd/md.yaml files. 

This file is a intermediate file and customer could not see them. In the customer facing file, the id/name will not (does not) show up if they are empty.

Reply:
I think we leave it if we don't make the template conditional. There are any edge cases we need to worry about by doing this right? Otherwise it might be worth making it conditional in the template.